Goal: Do not distribute the login.1 and su.1 Polish translations

Fixes: #491460

Status wrt upstream: Already applied

Index: shadow-4.1.1/man/pl/Makefile.am
===================================================================
--- shadow-4.1.1.orig/man/pl/Makefile.am
+++ shadow-4.1.1/man/pl/Makefile.am
@@ -20,7 +20,6 @@
 	grpconv.8 \
 	grpunconv.8 \
 	lastlog.8 \
-	login.1 \
 	login.defs.5 \
 	logoutd.8 \
 	newgrp.1 \
@@ -32,7 +31,6 @@
 	pwunconv.8 \
 	sg.1 \
 	shadow.5 \
-	su.1 \
 	suauth.5 \
 	useradd.8 \
 	userdel.8 \
